readme.rst

```diff
@@ -229,6 +229,34 @@ your doc files were placed (repository root or the ``docs`` directory):
    make install
    make html
 
+Build a PDF
+^^^^^^^^^^^
+
+Build a PDF locally with the following command:
+
+.. code-block:: none
+
+    make pdf
+
+PDF generation requires some system files. If these files are not found, a prompt will be presented and the generation will stop.
+
+On Linux, required packages can be installed with:
+
+.. code-block:: none
+
+    make pdf-prep-force
+
+.. note::
+    
+    When generating a PDF, the index page is considered a 'foreword' and will not be labelled with a chapter.
+
+.. important::
+    
+    When generating a PDF, it is important to not use additional headings before the ``toctree`` on the index page. Documents referenced by the
```

Comment:
It's not only on the index page - I just tested it.

If I have a subheading on any landing page and a toctree underneath:
```
Related links
-------------

Some links, just as mockup ...

.. toctree::
   :hidden:
   :maxdepth: 2

   /docs/quickstart
   ...
```

It'll show up fine in HTML:
![image](https://github.com/user-attachments/assets/08bc60d0-240a-4ff4-85b9-8f5c87c7e290)

But nested in PDF:
![image](https://github.com/user-attachments/assets/6d563b2c-4fec-4938-88ee-11c9af760bbe)

Reply:
So this means it needs to be marked as `rubric` everywhere we have subheadings and toctrees on the same page (or the toctree must be added before the subheadings). And if we use `rubric`, we lose the internal ToC.
I think that is acceptable, but it must be documented.

Reply:
Yeah this is true. I forgot about sub-index pages having toctrees.

Reply:
I added additional documentation about this to the readme.